src/librustdoc/html/static/rustdoc.css

```diff
@@ -955,6 +955,10 @@ a.test-arrow:hover{
 	position: absolute;
 	left: -23px;
 	top: 0;
+
+	/* The click event for this is defined on the document,
+	   so bubbling does not work. See https://github.com/rust-lang/rust/issues/83332 */
+	z-index: 10;
```

Comment:
Questions about this:
 * Does it go above the menu on mobile when the burger button has been pressed?
 * Does it go above the help popup (which appears after "?" has been pressed)?